extension-manifest-v3/src/background/telemetry/index.js

```diff
@@ -51,15 +51,24 @@ const loadStorage = async () => {
   return storage;
 };
 
-const getConf = async () => {
+const getConf = async (storage) => {
   const options = await store.resolve(Options);
 
+  if (!storage.installDate) {
+    saveStorage(storage, {
+      installDate:
+        options.installDate || new Date().toISOString().split('T')[0],
```

Comment:
That part still confuses me: why there is a fallback to options for installDate, but not for the random number. It is not obvious to me, why we need the fallback here? If storage.installDate is not present, how can options have it?

Reply:
If I understand correctly, options.installDate is migration code? And installRandom did not exist before.

Reply:
Comment added